config/schema/pathauto_pattern.schema.yml

```diff
@@ -0,0 +1,36 @@
+pathauto.pathauto_pattern.*:
```

Comment:
We could use a config_prefix to make this just pathauto.pattern.*, avoids the repetition.